interface/orders/types_edit.php

```diff
@@ -72,81 +71,152 @@ function recursiveDelete($typeid)
         "procedure_type_id = '$typeid'");
 }
 
+
 ?>
+<!DOCTYPE html>
 <html>
 <head>
-    <?php Header::setupHeader(['opener']); ?>
-    <title><?php echo $typeid ? xlt('Edit') : xlt('Add New'); ?><?php echo xlt('Order/Result Type'); ?></title>
+    <?php Header::setupHeader(['datetime-picker', 'jquery-ui', 'jquery-ui-base']);?>
+<script type="text/javascript"
+    src="<?php echo $webroot ?>/interface/main/tabs/js/include_opener.js"></script>
```

Comment:
remove above line and add opener to the setupHeader call above

Reply:
Done